pdns/recursordist/ext/Makefile.am

```diff
@@ -1,12 +1,15 @@
 SUBDIRS = \
 	yahttp \
-	json11
+	json11 \
+	probds
 
 DIST_SUBDIRS = \
 	yahttp \
-	json11
+	json11 \
+	probds
 
 EXTRA_DIST = \
 	luawrapper/include/LuaContext.hpp \
 	yahttp/LICENSE \
-	json11/LICENSE.txt incbin/incbin.h 
+	json11/LICENSE.txt incbin/incbin.h \
+	probds/hyperloglog.hpp
```

Comment:
hyperloglog.hpp doesn't seem to be used?

Reply:
No it's not. Well it was, but not anymore. We might use it in future for something, but in the meantime I'll remove it.